### DISCUSSION TITLE
Monika can't Read my Gifts :(

### DISCUSSION THREAD

**Discussion by Ilovecupcakes123** [Monika-After-Story/MonikaModDev#9168] [QA - tech support + general]
so I have been trying to give Monika gifts from the spritepacks but she can't read them. Maybe its because I accidentally changed the default app? I don't know...i am really desperate and I'm kinda sad that I can't give her gifts.

**Comment by WishStone-Dragon** [comment]
Well, anyone trying to help would need to know a lot more to start off. :3
Did gifting ever work for you? 
If it worked before, when did it stop working? 
What have you installed? Which packs?
How and where did you install it? 
Are there submods? 

Everything you can share can help pinpoint the problem.

**Comment by Ilovecupcakes123** [comment]
No, gifting has never worked.
I installed the latest pack (I think) which is the spritepacks combined? That's what it is called in the files at least.
I installed it on the Monika After Story Website 
I don't think there are submods...?

**Comment by Ilovecupcakes123** [comment]
and sorry for the late reply, I have a lot going on

**Comment by Ilovecupcakes123** [comment]
and sorry for the late reply, I have a lot going on

**Comment by lunulae** [comment]
Has Monika said she cannot read the file, or has she not responded to the gift even on reload?

If she has not responded to the file, you may need to make sure you are actually changing the _extension_ of the file to .gift, not just putting it at the end of the name (which would result in, for example, pinkshirt.gift.txt)

If she has _said_ she cannot read the file, first make sure you have restarted MAS before any gifting. If the problem persists, we may want a look at your game folders to make sure the installation of spritepacks was done correctly.

**Comment by Ilovecupcakes123** [comment]
She said that she couldn't read my gifts, and I do change the extension to ".gift". And yes, I restarted MAS before gifting Monika, but she still can't read it

**Comment by ThePotatoGuy** [comment]
are you sure that your file extensions are not hidden? As mentioned in the [FAQ entry](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#monika-doesnt-recognize-gifts)

**Reply by Ilovecupcakes123** [reply]
Yes, i don't think they're hidden

**Reply by Justformas** [reply]
Did you try following this troubleshooting installation for spritepacks guide here? https://github.com/Monika-After-Story/MonikaModDev/wiki/Adding-Sprite-Objects#installing-sprite-packs

**Comment by Error-game** [comment]
im also having this problem and i have checked all the steps is it because I have it open with note pad?
